src/openforms/utils/tests/html_assert.py

```diff
@@ -15,14 +15,14 @@ def assertTagWithTextIn(self, tag, text, document_str):
         """
         check for html tags and their content while ignoring tag attributes
         """
-        if not re.search(fr"<{tag}(?: [^>]*)?>\s*{text}", document_str):
+        if not re.search(rf"<{tag}(?: [^>]*)?>\s*{text}", document_str):
             self.fail(f"cannot find <{tag}..>{text} in: {document_str}")
 
     def assertNotTagWithTextIn(self, tag, text, document_str):
         """
         check for html tags and their content while ignoring tag attributes
         """
-        if re.search(fr"<{tag}(?: [^>]*)?>\s*{text}", document_str):
+        if re.search(rf"<{tag}(?: [^>]*)?>\s*{text}", document_str):
```

Comment:
this looks like an unexpected change - are you using a black version that's newer than the one pinned in the dev dependencies?